model/SearchEngine/Contract/BoolSearchInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2025 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoAdvancedSearch\model\SearchEngine\Contract;
+
+use oat\taoAdvancedSearch\model\SearchEngine\SearchResult;
+
+interface BoolSearchInterface
```

Comment:
Why do we need this? Why not a simple query using existing search method that matching the attribute that contains that item_uris in the tests?

We do not need this interface.